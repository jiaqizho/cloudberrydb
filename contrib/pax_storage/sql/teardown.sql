drop FUNCTION get_pax_aux_table;
-- start_ignore
drop schema if exists pax_test;
-- end_ignore
