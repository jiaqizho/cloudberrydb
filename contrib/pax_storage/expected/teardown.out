drop FUNCTION get_pax_aux_table;
